packages/styletron-engine-atomic/src/inject-style-prefixed.js

```diff
@@ -41,27 +42,12 @@ export default function injectStylePrefixed(
         continue;
       } else {
         // cache miss
-        let block = "";
-        const prefixed = prefix({[originalKey]: originalVal});
-        for (const prefixedKey in prefixed) {
-          const prefixedVal = prefixed[prefixedKey];
-          const prefixedValType = typeof prefixedVal;
-          if (prefixedValType === "string" || prefixedValType === "number") {
-            const prefixedPair = `${hyphenate(prefixedKey)}:${prefixedVal}`;
-            if (prefixedPair !== propValPair) {
-              block += `${prefixedPair};`;
-            }
-          } else if (Array.isArray(prefixedVal)) {
-            const hyphenated = hyphenate(prefixedKey);
-            for (let i = 0; i < prefixedVal.length; i++) {
-              const prefixedPair = `${hyphenated}:${prefixedVal[i]}`;
-              if (prefixedPair !== propValPair) {
-                block += `${prefixedPair};`;
-              }
-            }
-          }
-        }
-        block += propValPair; // ensure original prop/val is last (for hydration)
+        const cssDeclaration = `${hyphenatedProperty}:${((originalVal: any): string)};`;
+        const prefixed = prefix(cssDeclaration, hyphenatedProperty.length);
+        const block =
+          prefixed[prefixed.length - 1] === ";"
+            ? prefixed.slice(0, -1)
+            : prefixed;
```

Comment:
`stylis` guaranteed that the original declaration will be at the end of the prefixed declaration.

Reply:
Is this documented in stylis somewhere? It might be worth linking to the relevant part of stylis or at least mentioning it via a comment.

Reply:
I don't believe it is necessary.

The previous code was originally for adapting `inline-style-prefixer`, which doesn't guarantee the order of prefixed, while stylis has the same behavior as `autoprefixer`, which is "correct".